### DISCUSSION TITLE
AsyncAioPikaManager connect

### DISCUSSION THREAD

**Discussion by wangjianweiwei** [miguelgrinberg/python-socketio#738] [Q&A]
https://github.com/miguelgrinberg/python-socketio/blob/2f0d8bbd8c4de43fe26e0f2edcd05aef3c8c71f9/socketio/asyncio_aiopika_manager.py#L68-L76

A connection is built every time it is publish can this place be improved?

**Comment by miguelgrinberg** [comment]
Of course. This code was not written by myself, it is from a contributor. If you want to make improvements to it I'll be happy to merge them.

**Comment by wangjianweiwei** [comment]
Thanks. I'm glad to contribute the code.
